packages/medusa/src/api/routes/store/payment-collections/refresh-payment-session.ts

```diff
@@ -0,0 +1,36 @@
+import { IsInt, IsNotEmpty, IsString } from "class-validator"
+
+import { EntityManager } from "typeorm"
+import { PaymentCollectionService } from "../../../../services"
+
+export default async (req, res) => {
+  const data = req.validatedBody as AdminRefreshPaymentCollectionSessionRequest
+  const { id, session_id } = req.params
+
+  const paymentCollectionService: PaymentCollectionService = req.scope.resolve(
+    "paymentCollectionService"
+  )
+
+  const manager: EntityManager = req.scope.resolve("manager")
+  const payment_collection = await manager.transaction(
```

Comment:
nit(non-blocking): Camel case to follow general convention
```suggestion
  const paymentCollection = await manager.transaction(
```